Products/ZenHub/XmlRpcService.py

```diff
@@ -36,6 +37,11 @@ def __init__(self, dmd):
 
 
     def xmlrpc_sendEvent(self, data):
+        # Fix for ZEN-28005 to thwart XSS attacks from incoming events
+        for f in data:
+            if f in ("eventClassKey", "component", "summary", "device", "eventClass"):
+                if data[f] is not None and len(data[f]) > 0:
+                    data[f] = cgi.escape(data[f])
```

Comment:
there seems to be a few sendevent methods -- is this the specific one you are referring to? https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenEvents/SendEvent.py